ui/src/kapacitor/components/RuleHeader.js

```diff
@@ -110,31 +110,31 @@ export const RuleHeader = React.createClass({
           onKeyDown={e => this.handleEditName(e, rule)}
           onBlur={() => this.handleEditNameBlur(rule)}
           placeholder="Name your rule"
+          spellCheck={false}
+          autoComplete={false}
         />
-      : <h1
-          className="page-header__title page-header--editable kapacitor-theme"
-          onClick={this.toggleEditName}
-          data-for="rename-kapacitor-tooltip"
-          data-tip="Click to Rename"
-        >
-          {rule.name}
-          <span className="icon pencil" />
-          <ReactTooltip
-            id="rename-kapacitor-tooltip"
-            delayShow={200}
-            effect="solid"
-            html={true}
-            offset={{top: 2}}
-            place="bottom"
-            class="influx-tooltip kapacitor-tooltip place-bottom"
-          />
-        </h1>
+      : <div className="page-header__left">
+          <h1
+            className="page-header__title page-header--editable kapacitor-theme"
+            onClick={this.toggleEditName}
+            data-for="rename-kapacitor-tooltip"
+            data-tip="Click to Rename"
+          >
+            {rule.name}
+            <span className="icon pencil" />
```

Comment:
I guess since we're rendering this through React, we don't need to worry so much about closing all our tags. We should verify that the Virtual DOM handles these cases by adding a closing tag.

I think if this self closing tag gets through to the client, that the browser will interpret this as an open tag without its closing pair. And everything that follows will be a child instead of a sibling.